src/logging/event.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::Result;
+use slog::{OwnedKVList, KV};
+
+use super::ser::KubewardenLogSerializer;
+use serde_json::json;
+
+pub(crate) fn new(
+    rinfo: &slog::Record,
+    logger_values: &OwnedKVList,
+) -> Result<serde_json::Map<String, serde_json::Value>> {
+    let level: String = String::from(match rinfo.level() {
+        slog::Level::Debug => "debug",
+        slog::Level::Info => "info",
+        slog::Level::Warning => "warning",
+        slog::Level::Error => "error",
+        _ => "error",
```

Comment:
This flattens `slog::Level::Critical` and `slog::Level::Trace` to `error` which I think is fine right now. However I would be explicit here removing the catch all. This ensures that if a new enum entry is added in the future, the build will fail, giving us the chance to implement different logic, instead of using `_` as a muting sink.

Reply:
Done, makes a lot of sense